web/ui/react-app/src/pages/targets/ToggleMoreLess.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { Button } from 'reactstrap';
+
+interface ToggleProps {
```

Comment:
Usually our naming for props is `<component name>Props`, so `ToggleMoreLessProps` here.
```suggestion
interface ToggleMoreLessProps {
```